binr/r2pm/r2pm

```diff
@@ -524,7 +524,9 @@ r2pm_doc() {
 }
 
 r2pm_install() {
-	r2pm_update
+	if [ "${R2PM_OFFLINE}" = 0 ]; then
```

Comment:
move this check into the r2pm_update() function. and just do a oneliner like this:

`[ "${R2PM_OFFLINE}" != 0 ] && return`